docs/clients/arming_handler.rst

```diff
@@ -144,9 +144,9 @@ Type ID 86 | Arming Handler
         || 11    || manual_arming_throttle_source      || get, set, save || uint8    ||                         || The :ref:`throttle source <throttle_sources>` used as its armed throttle source for manual arming. The default manual throttle source is Unknown. In order to use manual arming, a throttle source must be set first. See row below for each state: |
         ||       ||                                    ||                ||          ||                         || 0 = Unknown: Default configuration. There will be no throttle source if you manually arm, so don't leave this as Unknown if you are manuall arming.                                                                                                 |
-        ||       ||                                    ||                ||          ||                         || 1 = :ref:`Hobby <manual_hobby>`                                                                                                                                                                                                                     |
-        ||       ||                                    ||                ||          ||                         || 2 = :ref:`DroneCAN <manual_dronecan>`                                                                                                                                                                                                               |
-        ||       ||                                    ||                ||          ||                         || 3 = :ref:`IQUART <manual_iquart>`                                                                                                                                                                                                                   |
+        ||       ||                                    ||                ||          ||                         || 1 = :ref:`Hobby <hobby_protocol>`                                                                                                                                                                                                                   |
+        ||       ||                                    ||                ||          ||                         || 2 = :ref:`DroneCAN <dronecan_protocol>`                                                                                                                                                                                                             |
+        ||       ||                                    ||                ||          ||                         || 3 = :ref:`IQUART <uart_messaging>`                                                                                                                                                                                                                  |
```

Comment:
uart_messaging was the existing reference for the UART Messaging section, I left it as that since it was accurate enough